.github/workflows/workflow_dispatch_failure_test.yaml

```diff
@@ -0,0 +1,23 @@
+name: Workflow Dispatch Failure Tests 2a34f8b1-41e4-4bcb-9bbf-7a74e6c482f7
```

Comment:
Would it be better to use a different name than UUID here?

Reply:
The theory is that, whatever name we pick, may clash with a workflow name of a user and then that workflow always gets declined. Including a UUID reduces the chance of a clash essentially to zero which is why it is useful in this case